align/compiler/write_verilog_lef.py

```diff
@@ -278,6 +278,10 @@ def generate_lef(name, values, available_block_lef,
             size = int(values["nfin"])
             if 'nf' in values.keys():
                 size=size*int(values["nf"])
+            if 'm' in values.keys():
+                size=size*int(values["m"])
+            if 'M' in values.keys():
+                size=size*int(values["m"])
```

Comment:
Please check coverage on this piece of code. We are checking for 'M' but then querying values['m']. I believe this should result in a KeyError. It may be a good idea to convert all keys to upper case or lower case as a pre-processing step if the SPICE spec we are following says everything is case-insensitive.

Reply:
converting all value parameters to lower case () in basic_element.py .